pkgs/applications/networking/ids/suricata/default.nix

```diff
@@ -53,7 +53,7 @@ stdenv.mkDerivation rec {
     jansson
     libbpf
     libcap_ng
-    libelf
+    elfutils
```

Comment:
Can you move the entry up so that the alphabetical order is preserved?
Please also align it with the input occurrences in the parameter attributes, basically between `makeWrapper` and `file`.

Reply:
Sure, if you like.